variation-commons-mongodb/src/main/java/uk/ac/ebi/eva/commons/mongodb/services/VariantWithSamplesAndAnnotationsService.java

```diff
@@ -268,6 +269,21 @@ public long countTotalNumberOfVariants() {
         return variantRepository.findDistinctChromosomes();
     }
 
+    public List<VariantMongo> findByRegionAndOtherBeaconFilters(Region startRange, Region endRange,
+                                                                List<VariantRepositoryFilter> filters,
+                                                                Pageable pageable) {
+        return variantRepository.findByRegionAndOtherBeaconFilters(startRange, endRange, filters, pageable);
+    }
+
+    public Long countByRegionAndOtherBeaconFilters(Region startRange, Region endRange,
+                                                      List<VariantRepositoryFilter> filters) {
+        return variantRepository.countByRegionAndOtherBeaconFilters(startRange, endRange, filters);
+    }
+
+    public List<VariantSourceMongo> findAllVariantSourcesForBeacon() {
```

Comment:
Shouldn't this method be part of the service that allows to query variant sources instead of variants?

Reply:
I will change it to `VariantSourceService.java` :+1:

Reply:
Fixed :)